.gitignore

```diff
@@ -21,6 +21,6 @@ index.html
 sftp-config.json
 npm-debug.log
 version.txt
-
+mdHelp.html
```

Comment:
As all `index.html` files have been added in `.gitignore`; I assume it is only appropriate to add `mdHelp.html` too.